dev/com.ibm.ws.springboot.fat20_fat/fat/src/com/ibm/ws/springboot/support/fat/utility/SpringBootUtilityThinTest.java

```diff
@@ -360,13 +360,15 @@ public void testDefaultHostWithAppPortRunLibertyUberJarWithSSL() throws Exceptio
 
         String line = null;
         try (BufferedReader reader = new BufferedReader(new InputStreamReader(proc.getInputStream()))) {
+            readTimeout(reader);
```

Comment:
You should assign the result of `readTimeout` call to `line` and not read a new line after each call.  Otherwise you are skipping every other line and also seems like it would hang on the call to `line = reader.readLine();` if we ran out of lines to read.